src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -1,27 +1,33 @@
 <script setup>
-import { ref } from 'vue'
+import { computed, ref } from 'vue'
+import { MaInput } from "@mobileaction/action-kit"
+import { MaBadge } from "@mobileaction/action-kit"
 
+const checkedNumbers = ref([])
+const excludeWords = ref('is, a, an, the') // default words to exclude
 const sentence = ref('')
```

Comment:
```suggestion
const inputText = ref('')
```

Changing the name from sentence to inputText might be more descriptive.